public/apps/configuration/utils/display-utils.tsx

```diff
@@ -31,14 +31,14 @@ export function renderTextFlexItem(header: string, value: string) {
   );
 }
 
-export function displayBoolean(bool: boolean) {
+export function displayBoolean(bool: boolean | undefined) {
```

Comment:
You can use bool? to indicate boolean or undefined.

Reply:
Tried it but received TS errors complaining about bool not being a type.

Reply:
I think you meant capitalized Boolean. Tried it and it worked. Will update.